applications/CompressiblePotentialFlowApplication/tests/potential_flow_test_factory.py

```diff
@@ -72,6 +72,15 @@ def test_SmallLiftJumpTest(self):
 
             kratos_utilities.DeleteTimeFiles(".")
 
+    def test_EmbeddedCircleNoWake(self):
+        settings_file_name = "embedded_circle_no_wake_parameters.json"
+        work_folder = "embedded_test"
+
+        with WorkFolderScope(work_folder):
+            self._runTest(settings_file_name)
+
+            kratos_utilities.DeleteTimeFiles(".")
```

Comment:
I am curious, does it still write the time files?
I thought I had disabled this globally

Reply:
You are right, it does not write it anymore!